helm/basex-validator/templates/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "basex-validator.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "basex-validator.name" . }}
+    helm.sh/chart: {{ include "basex-validator.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "basex-validator.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "basex-validator.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
```

Comment:
can usually give a generic name like `app` to this container, because it's namespaced inside a Pod named `basex-validator--test` already